src/bssl.rs

```diff
@@ -14,10 +14,25 @@
 
 use {c, error};
 
-pub fn map_result(bssl_result: c::int) -> Result<(), error::Unspecified> {
-    match bssl_result {
-        1 => Ok(()),
-        _ => Err(error::Unspecified),
+/// A `c::int` returned from a foreign function containing **1** if the function was successful
+/// or **0** if an error occurred. This is the convention used by BoringSSL.
```

Comment:
Let's remove the "This is the convention used by BoringSSL." I don't want to conflate *ring* with BoringSSL.